src/pages/Contributing/Designers/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Definition of Done for symbols'
+order:  13
+---
+
+> Mozaic is a worldwide source of truth when it comes to design for Adeo and Leroy Merlin's products. That is why you need to use general guidelines when you contribute to Mozaic. We made some rules to follow when designing new symbols.
+
+This **Definition of Done** page has been divided in 10 steps.
+
+## 1. The pattern doesn't exist in Mozaic
+I checked that the pattern is not in another library :
```

Comment:
I checked that the pattern is not in another library or present in a current issue :